### ISSUE TITLE
FR: Home link in new frame when opening in new tab

### ISSUE THREAD

**Issue by douglasawh** [NagiosEnterprises/nagioscore#329]
While the home option is there of you stay in the original frame, it is absent when you move to a new frame.

Additional detail at https://support.nagios.com/forum/viewtopic.php?t=42445&start=9999#bottom

**Comment by ericloyd** [comment]
Personally, I would consider this "works as intended" since opening frame targets as separate tabs means you lose the menu structure, no matter what web page you're on.

**Comment by rcknag** [comment]
Why not just use a function to detect if in a frame, and echo link if not? http://stackoverflow.com/questions/326069/how-to-identify-if-a-webpage-is-being-loaded-inside-an-iframe-or-directly-into-t

**Comment by ericloyd** [comment]
I'm really not sure I see this as being beneficial to change the UI for everyone.  I mean, I open a LOT of frames in XI and Core to do things with service definitions and other things.  Generally, they're opened from another tab that's on the home page or view page or whatever.  When I'm done with the tab, I close it.  I don't need to return home from a sub tab if I've opened it as a tab; I did so for a temporary reason.

Adding complexity that simply adds a home button to every page also means that any tool screen-scraping things off of these pages needs to be adjusted to account for the new button.
